docs/publish/macos.md

```diff
@@ -11,4 +11,20 @@ The command can be run on macOS only.
 
 ## `flet build macos`
 
-Creates a macOS application bundle from your Flet app.
\ No newline at end of file
+Creates a macOS application bundle from your Flet app.
+
+### Specify output directory
+
+If you specify the build directory then the directory acts like a cache, speeding up subsequent builds.
+
+```bash
+flet build -o ./build macos
+```
```

Comment:
A default output directory is used when one is not specified: `build/macos` when using `flet build macos`

Reply:
I had just been using `-o ./build`. I didn't know about the default. That can be removed from these changes.